### PR TITLE
Show-update-destroy Category

### PR DESCRIPTION
**Pull request by phuongbk**

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -17,6 +22,29 @@ def create
     end
   end
 
+  def edit
+    @category = Category.find params[:id]
+  end
+
+  def update
+    @category = Category.find params[:id]
```
Comment (dieunb) [inline app/controllers/admin/categories_controller.rb]: `@category = Category.find params[:id]` is many duplication. You can use `before_action` to `set_category` for some actions.

File: db/seeds.rb [hunk 2]
```diff
@@ -6,6 +6,24 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+# User.create! name: "admin", email: "adminzzz@gmail.com",
```
Comment (dieunb) [inline db/seeds.rb]: if you don't use, you can remove it.

File: app/views/admin/categories/_edit_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<%= form_for [:admin, @category] do |f| %>
```
Comment (VoTaiTri) [inline app/views/admin/categories/_edit_form.html.erb]: this file is duplicate with file pp/views/admin/categories/_form.html.erb
try to remove this file if you don't need

File: app/views/admin/categories/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      <h1>
+        <%= @category.name %>
+      </h1>
+      <p>
+        <%= @category.description %>
+      </p>
+      <p>
+        <%= t "info.created_time" %><%= @category.created_at %>
+      </p>
+       [:edit, :admin, @category]<p>
```
Comment (VoTaiTri) [inline app/views/admin/categories/show.html.erb]: what for?

**Pull request merged**